src/pages/achievement/control/subcomponents/editorTools/AchievementTemplate.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  AchievementAbility,
+  AchievementItem,
+  AchievementModalItem
+} from '../../../../../commons/achievement/AchievementTypes';
+
+export const modalTemplate: AchievementModalItem = {
+  modalImageUrl:
+    'https://source-academy-assets.s3-ap-southeast-1.amazonaws.com/images/sourceAcademy%40x2.png',
```

Comment:
Please refactor the S3 URL into src/commons/util/Constants.ts under Links, the Game team will follow suit afterwards.

Reply:
Done ca57ca4